### DISCUSSION TITLE
flink-connector-oracle-cdc.version：2.4.0监听了oracle所有的schema的表数据变更？而不是我指定的schema

### DISCUSSION THREAD

**Discussion by jasonhewg** [apache/flink-cdc#2466] [Q&A]
Flink version
flink.version：1.14.2

Flink CDC version
flink-connector-oracle-cdc.version：2.4.0

Database and its version
oracle：19c

Minimal reproduce step
SourceFunction<String> sourceOracle = OracleSource.<String>builder()
                .hostname("172.16.131.xxx")
                .port(1521)
                .database("ORCL")
                .schemaList("FILE_MINIO")
                .tableList("FILE_MINIO.FILE_DETAIL")
                .username("FILE_MINIO")
                .password("xxx")
                .debeziumProperties(properties)
                .deserializer(new JsonDebeziumDeserializationSchema())
                .build();

通过上面的OracleSource我指定了具体需要监控的某一个schema，如：FILE_MINIO，但RelationalSnapshotChangeEventSource输出日志显示，会监听oracle所有的schema，部分日志：
      INFO [debezium-oracleconnector-oracle_logminer-change-event-source-coordinator][192]io.debezium.relational.RelationalSnapshotChangeEventSource : Adding table ORCL.GL_YDW.A202_XPHARMA_USER_DETAIL to the list of capture schema tables
2023-09-06 23:08:07.559
      INFO [debezium-oracleconnector-oracle_logminer-change-event-source-coordinator][192]io.debezium.relational.RelationalSnapshotChangeEventSource : Adding table ORCL.GROUP_YDW.XPHARMA_GROUPING_USER_BU_SYNC to the list of capture schema tables
2023-09-06 23:08:07.560
      INFO [debezium-oracleconnector-oracle_logminer-change-event-source-coordinator][192]io.debezium.relational.RelationalSnapshotChangeEventSource : Adding table ORCL.EBR_SF_YDW.A202_XPHARMA_MAT_CUSTOM_MAN to the list of capture schema tables
2023-09-06 23:08:07.560
      INFO [debezium-oracleconnector-oracle_logminer-change-event-source-coordinator][192]io.debezium.relational.RelationalSnapshotChangeEventSource : Adding table ORCL.SF_YDW.A202_XPHARMA_AUTOCODE_SN to the list of capture schema tables
2023-09-06 23:08:07.560
      INFO [debezium-oracleconnector-oracle_logminer-change-event-source-coordinator][192]io.debezium.relational.RelationalSnapshotChangeEventSource : Adding table ORCL.NACOS_CONFIG.USERS to the list of capture schema tables
2023-09-06 23:08:07.560
      INFO [debezium-oracleconnector-oracle_logminer-change-event-source-coordinator][192]io.debezium.relational.RelationalSnapshotChangeEventSource : Adding table ORCL.GL_YDW.A202_XPHARMA_NL_PRINT_TEMPLATE to the list of capture schema tables
2023-09-06 23:08:07.560
      INFO [debezium-oracleconnector-oracle_logminer-change-event-source-coordinator][192]io.debezium.relational.RelationalSnapshotChangeEventSource : Adding table ORCL.EBR_SF_YDW.A202_XPHARMA_MAT_ENCRYPTION to the list of capture schema tables
2023-09-06 23:08:07.560

**Comment by lfift** [comment]
将debezium这个属性设置为true，database.history.store.only.captured.tables.ddl
